README.md

```diff
@@ -26,105 +26,106 @@
+💾 [Lovely forks](https://github.com/musically-ut/lovely-forks#lovely-forks)  
+💾 [npmhub](https://github.com/npmhub/npmhub)  
+💾 [Octotree](https://github.com/buunguyen/octotree/#octotree)  
+💾 [OctoLinker](https://github.com/OctoLinker/OctoLinker)  
+💾 [Refined GitHub](https://github.com/sindresorhus/refined-github)  
+💾 [ZenHub](https://www.zenhub.io/)
 
 ## Available Syntax Highlighting Themes ([Demo](https://stylishthemes.github.io/GitHub-Dark/))
 
-| Theme                      | GitHub | CodeMirror | Jupyter  |
-|----------------------------|:------:|:----------:|:--------:|
-| Ambiance                   |   ◻️    |     🔹      |          |
-| Base16 Ocean Dark          |        |     🔹      |     🔸    |
-| Chaos                      |   ◻️    |            |          |
-| Clouds Midnight            |   ◻️    |            |          |
-| Cobalt                     |   ◻️    |     🔹      |          |
-| Dracula                    |        |     🔹      |     🔸    |
-| GitHub Dark                |   ◻️    |            |     🔸    |
-| Idle Fingers               |   ◻️    |            |     🔸    |
-| Kr Theme                   |   ◻️    |            |          |
-| Material                   |        |     🔹      |          |
-| Merbivore                  |   ◻️    |            |          |
-| Merbivore Soft             |   ◻️    |            |          |
-| Mono Industrial            |   ◻️    |            |          |
-| Mono Industrial Clear      |   ◻️    |            |          |
-| Monokai                    |   ◻️    |     🔹      |     🔸    |
-| Monokai Spacegray Eighties |   ◻️    |     🔹      |     🔸    |
-| Obsidian                   |   ◻️    |            |     🔸    |
-| One Dark                   |   ◻️    |     🔹      |          |
-| Pastel on Dark             |   ◻️    |     🔹      |     🔸    |
-| Solarized Dark             |   ◻️    |     🔹      |     🔸    |
-| Terminal                   |   ◻️    |            |          |
-| Tomorrow Night             |   ◻️    |            |     🔸    |
-| Tomorrow Night Blue        |   ◻️    |            |     🔸    |
-| Tomorrow Night Bright      |   ◻️    |     🔹      |     🔸    |
-| Tomorrow Night Eighties    |   ◻️    |     🔹      |     🔸    |
-| Twilight (**default**)     |   ◻️    |     🔹      |     🔸    |
-| Vibrant Ink                |   ◻️    |     🔹      |          |
-
-* Support for [Codemirror](https://codemirror.net/demo/theme.html) and [Jupyter notebook syntax highlighting](https://github.com/sujitpal/statlearning-notebooks/blob/master/src/chapter2.ipynb) themes as listed above.
-* Please provide a pull request if you have or want to create a missing theme.
+| Theme                      |       GitHub       |     CodeMirror     |      Jupyter       |
+|----------------------------| :----------------: | :----------------: | :----------------: |
+| Ambiance                   | :heavy_check_mark: | :heavy_check_mark: |                    |
```

Comment:
Please use unicode characters so it looks good outside GitHub too

Reply:
:sob: